build/makelib/golang.mk

```diff
@@ -162,12 +165,17 @@ go.fmt:
 
 go.validate: go.vet go.fmt
 
-.PHONY: go.vendor
-go.vendor $(GO_VENDOR_DIR)/vendor.stamp: $(GLIDE)
+$(GLIDE_LOCK): $(GLIDE) $(GLIDE_YAML)
 	@echo === updating vendor dependencies
 	@mkdir -p $(GLIDE_HOME)
+	@$(GLIDE) update --strip-vendor
+	@touch $@
```

Comment:
since $(GLIDE_YAML) is a pre-req for the rule it would have to have a newer timestamp than $(GLIDE_LOCK) for this recipe to run. `$@` refers to the target i.e. $(GLIDE-LOCK) we touch it just to be safe its newer than $(GLIDE_YAML)